backend/utils/generic_errors.py

```diff
@@ -0,0 +1,6 @@
+class SlideRequestError(Exception):
```

Comment:
let's make a base exception that will be the parent for all of out custom exceptions